babybuddy/settings/base.py

```diff
@@ -26,6 +26,7 @@
     "api",
     "babybuddy",
     "core",
+    "taggit",
```

Comment:
I wonder -- can we remove this to prevent adding the default model (i.e., what shows up in the Django admin interface under "Taggit") since we have out own custom model?

Unclear if that is what is meant by this note --

> Note: Including ‘taggit’ in settings.py INSTALLED_APPS list will create the default django-taggit and “through model” models. If you would like to use your own models, you will need to remove ‘taggit’ from settings.py’s INSTALLED_APPS list.

From https://django-taggit.readthedocs.io/en/latest/custom_tagging.html#using-a-custom-tag-or-through-model

Reply:
Ah yes! That seems to work. I had problems removing the app before, but cannot recall what that was. Now, with all custom models it seems to work. Could not spot any issues. Good one! :+1: